code/modules/antagonists/revolution/revolution.dm

```diff
@@ -14,6 +14,8 @@
 	var/datum/team/revolution/rev_team
 	///when this antagonist is being de-antagged, this is why
 	var/deconversion_reason
+	/// What sound should we play for becoming this brand of revolutionary?
+	var/internationale = 'sound/ambience/antag/viva_short.ogg'
```

Comment:
Rename this variable to `gain_sound` or similar to signify its actual meaning.

Reply:
yeet